ngsi_event_broker/script/build/jenkins.sh

```diff
@@ -133,8 +133,8 @@ build)
 	make clean lint-report test-report coverage-report
 
 	# Copy reports with paths relative to $WORKSPACE root
-	sed 's#\./src/#ngsi_event_broker/src/#' $COVERAGE_REPORT_DIR/lcov.info > $PROJECT_DIR/jenkins-lcov.info
-	sed 's#filename="src/#filename="ngsi_event_broker/src/#' $COVERAGE_REPORT_DIR/cobertura-coverage.xml > $PROJECT_DIR/jenkins-cobertura-coverage.xml
+	#sed 's#\./src/#ngsi_event_broker/src/#' $COVERAGE_REPORT_DIR/lcov.info > $PROJECT_DIR/jenkins-lcov.info
+	sed 's#filename="src/#filename="/var/develenv/jenkins/jobs/metrics-queue-consumer/workspace/workspace/ngsi_event_broker/src/#' $COVERAGE_REPORT_DIR/cobertura-coverage.xml > $PROJECT_DIR/jenkins-cobertura-coverage.xml
 
```

Comment:
This change is not needed: relative paths are used for Jenkins-only reports; absolute paths are left intact for SonarQube